hq/app/logging/Cloudwatch.scala

```diff
@@ -69,6 +69,22 @@ object Cloudwatch extends Logging {
     putMetric("SecurityHQ", "Vulnerabilities", Seq(("GcpProject", project),("DataType", dataType.toString)), value)
   }
 
+  def putIamRemovePasswordSuccessMetric(): Unit = {
+    putMetric("SecurityHQ", "IamRemovePasswordExecution", Seq(("Status", "Success")), 1)
```

Comment:
Minor thing but might be nice to put some of these strings in `val`s or an enumeration/type.